packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -850,15 +903,25 @@ class Scaffold extends StatefulWidget {
   ///  * [showModalBottomSheet], which displays a modal bottom sheet.
   final Widget bottomSheet;
 
-  /// Whether the [body] (and other floating widgets) should size themselves to
-  /// avoid the window's bottom padding.
+  /// This flag is deprecated, please use resizeToAvoidBottomInset
+  /// instead.
+  ///
+  /// Originally the name referred [MediaQueryData.padding]. Now it refers
+  /// [MediaQueryData.viewInsets], so using [resizeToAvoidBottomInset]
+  /// should be clearer to readers.
+  @deprecated
```

Comment:
Looking at the rest of the codebase, `@Deprecated('reason')` is the Dart annotation and `@deprecated` is the dartdoc annotation.

Reply:
Good point, updated.